development/language/usage.rst

```diff
@@ -114,6 +114,19 @@ To use the language entry with the key ``MY_KEY`` in a template file, just write
 ``{L_MY_KEY}`` in the template (phpBB syntax) or ``{{ lang('MY_KEY') }}`` (new
 twig syntax).
 
+If the language entry is going to be used inside of JavaScript, it must be properly
+escaped. This is easy to achieve by using ``{LA_MY_KEY}`` in the template (phpBB 
+syntax) or ``{{ lang('MY_KEY')|e('js') }}`` (new twig syntax).
+
+.. note::
+
+    When using language extries in the JavaScript context with the new twig syntax,
+    the following methods are possible:
+    ``{{ lang('MY_KEY')|escape('js') }}`` Twig's native JavaScript context escape filter.
```

Comment:
New line before the list and then start all list items with `* ` (adding 4 space before the `*` will put the list in the note